backend/apps/bookings/models.py

```diff
@@ -49,9 +58,50 @@ def cancel_booking(self):
         self.save()
         return True
 
-    def booking_end_time(self, duration_in_minutes=60):
-        return self.session_date + timedelta(minutes=duration_in_minutes)
+    def booking_duration(self):
+        return self.session_end_time - self.session_date
```

Comment:
```suggestion
    def booking_duration(self):
        if not self.session_end_time:
              return timedelta(0)
        return self.session_end_time - self.session_date
```

A check to ensure that session_end_time is not empty.